src/main/scala/is/hail/expr/ir/Pretty.scala

```diff
@@ -46,6 +46,7 @@ object Pretty {
             case I64(x) => x.toString
             case F32(x) => x.toString
             case F64(x) => x.toString
+            case Str(x) => s"'$x'"
```

Comment:
sorry, can we use double quotes? Scala doesn't recognize single quotes as strings, so we'd need double quotes to make things pastable